.travis.yml

```diff
@@ -30,7 +30,7 @@ stages:
 
 stage: test
 install:
-  - if [[ -n "$ONIGURUMA" ]]; then tags="$tags oniguruma"; fi; go get -v -t --tags "$tags" ./...
+  - if [[ "${ONIGURUMA}" -gt 0 ]]; then export tags="${tags} oniguruma"; CGO_ENABLED=1; fi; go get -v -t -tags "${tags}" ./...
```

Comment:
I _believe_ if you want `CGO_ENABLED` to work you'll either have to export it into the environment or put it on the command-line with the `go …` command, e.g., `CGO_ENABLED=1 go get …`.

Reply:
Good catch! Although it works without export as well